streamlit_drawable_canvas/frontend/src/lib/rect.ts

```diff
@@ -48,8 +49,8 @@ class RectTool extends FabricTool {
       top: this.currentStartY,
       originX: "left",
       originY: "top",
-      width: pointer.x - this.currentStartX,
-      height: pointer.y - this.currentStartY,
+      width: this._minLength,
+      height: this._minLength,
```

Comment:
Fix bug. Without this, the user can create zero-sized rectangles with single left-click. (The `circle.ts` has no such a problem because the code had been correctly implemented in previous PR.)